packages/flutter/test/widgets/semantics_9_test.dart

```diff
@@ -139,14 +137,14 @@ void main() {
 }
 
 class BoundaryBlockSemantics extends SingleChildRenderObjectWidget {
-  const BoundaryBlockSemantics({ Key key, Widget child }) : super(key: key, child: child);
+  const BoundaryBlockSemantics({ Key? key, Widget? child }) : super(key: key, child: child);
 
   @override
   RenderBoundaryBlockSemantics createRenderObject(BuildContext context) => RenderBoundaryBlockSemantics();
 }
 
 class RenderBoundaryBlockSemantics extends RenderProxyBox {
-  RenderBoundaryBlockSemantics({ RenderBox child }) : super(child);
+  RenderBoundaryBlockSemantics({ RenderBox? child }) : super(child);
```

Comment:
Can the child be required here too?

Reply:
This child is not used at all by the test code, so I just removed it.